src/components/Camera/StandardCamera.js

```diff
@@ -90,13 +93,30 @@ const StandardCamera = ( ): Node => {
     navigation.navigate( "ObsEdit" );
   };
 
-  const renderCameraOptionsButtons = icon => (
-    <Avatar.Icon
-      size={40}
-      icon={icon}
-      style={{ backgroundColor: colors.gray }}
-    />
-  );
+  const renderCameraOptionsButtons = icon => {
+    let accessibilityLabel = "";
+    switch ( icon ) {
+      case "flash":
+        accessibilityLabel = "The flash is turned on. Press here to disable it.";
```

Comment:
These should just come from the translation files. I would add strings like `flash-button-label-flash = The flash is turned on. Press here to disable it.` I'd be fine with doing that in this PR, but also happy to do a more systemic task. Where else where you seeing string literals in the components like this?

Reply:
OK, I have added translations also to the other instances I found with accessibility labels. However, now my tests did not pass anymore because i18n is mocked in the tests. So, I switched to finding the button by id instead. But I think I would prefer to be able to find the buttons by label in English.